### DISCUSSION TITLE
404 File Not Found Error

### DISCUSSION THREAD

**Discussion by usertakenusernameisnotavailable** [community/community#31147] [New to GitHub]
Hello,

I'm an amateur in terms of web development and Github.

Recently I have started migrating my web development projects from Replit to Github, however, when I used Jekyll to jumpstart my web project, it showed a 404 Error on https://usertakenusernameisnotavailable.github.io/calculator/.

What does it mean by "If this is your site, make sure that the filename case matches the URL.
For root URLs (like http://example.com/) you must provide an index.html file.:

**Comment by usertakenusernameisnotavailable** [comment]
SORRY NVM FOUND ANSWER

**Reply by LiriDek** [reply]
Just had the same issue right now and it's fixed because i wrote the first letter of index with a capital letter lol.

NOT Index.html but index.html.

Did you had the same issue ?

**Reply by queenofcorgis** [reply]
Happy to hear you found the answer @usertakenusernameisnotavailable - mind sharing it with the community in case someone runs into a similar situation?

Thanks @LiriDek for sharing how you resolved your issue ❤️ :octocat:

**Reply by 3kh0** [reply]
@usertakenusernameisnotavailable make sure you mark this as the answer!
